src/wallet/utils/getWindowDimensions.ts

```diff
@@ -0,0 +1,35 @@
+import type { WindowSizes } from '../types';
+
+const popupSizes: WindowSizes = {
+  sm: { width: '24.67vw', height: '30.83vw' },
```

Comment:
`vw` are we sure we want this? I feel a bit hesitant of using `vw`. Like how this behavies on mobile?

Reply:
added some screenshots on window sizing on desktop and mobile

Reply:
I think on mobile it will always open in a new tab? I don't think there's popup windows on most mobile browsers

Reply:
I'm hoping what will happen on mobile is it will ignore the `windowFeatures` parameter and just open in a new tab, but no way to emulate this mobile behavior on Chrome devtools I don't think